app/src/main/java/org/mozilla/focus/autocomplete/AutocompleteListFragment.kt

```diff
@@ -215,7 +215,7 @@ open class AutocompleteListFragment : Fragment() {
             }
         }
 
-        override fun onViewRecycled(holder: RecyclerView.ViewHolder?) {
+        override fun onViewRecycled(holder: RecyclerView.ViewHolder) {
             if (holder is DomainViewHolder) {
                 holder.checkBoxView.setOnCheckedChangeListener(null)
             }
```

Comment:
The changes in this file seem to be unrelated? :) But they are in a separate commit, so no problem :)